### PR TITLE
Add pixi-filters@1.0.6 /w npm auto-update

### LINKED ISSUE #9809: [Request] pixi-filters
pixi-filters
https://github.com/pixijs/pixi-filters
http://opensource.org/licenses/MIT
http://www.pixijs.com/

### PR DESCRIPTION
**Pull request by clairetsai818**
close #9809, cc @bigtimebuddy

Pull request for issue: #9809 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/pixijs/pixi-filters/tree/v1.0.6
 * NPM package url (optional): https://www.npmjs.com/package/pixi-filters
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 34
   - Count of watchers: 8
   - Count of forks: 6
 * NPM download stats (optional):
   - Downloads in the last day: 0 
   - Downloads in the last week: 29
   - Downloads in the last month: 150

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@bigtimebuddy thank you for helping us review this PR :+1:

File: ajax/libs/pixi-filters/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "pixi-filters",
+  "version": "1.0.6",
+  "description": "Optional display filters to work with Pixi.js",
+  "author": "Mat Groves",
+  "filename": "filters.min.js",
+  "homepage": "http://goodboydigital.com/",
```
Comment (PeterDaveHello) [inline ajax/libs/pixi-filters/package.json]: @bigtimebuddy is this url still its hompage? Or maybe we should use `https://pixijs.github.io/pixi-filters/docs/` instead?

Comment (bigtimebuddy) [inline ajax/libs/pixi-filters/package.json]: Either the docs or the main github page. We don't actually have a homepage for the filters.

Comment (PeterDaveHello) [inline ajax/libs/pixi-filters/package.json]: @clairetsai818 let's update this part.

Comment (clairetsai818) [inline ajax/libs/pixi-filters/package.json]: @PeterDaveHello I've updated this part, please help me review this. Thanks.

**Pull request merged**